composer.json

```diff
@@ -20,15 +26,15 @@
     "docs": "http://andersao.github.io/l5-repository"
   },
   "require": {
-    "illuminate/http": "~5.0|~6.0|~7.0",
-    "illuminate/config": "~5.0|~6.0|~7.0",
-    "illuminate/support": "~5.0|~6.0|~7.0",
-    "illuminate/database": "~5.0|~6.0|~7.0",
-    "illuminate/pagination": "~5.0|~6.0|~7.0",
-    "illuminate/console": "~5.0|~6.0|~7.0",
-    "illuminate/filesystem": "~5.0|~6.0|~7.0",
+    "illuminate/http": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/config": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/support": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/database": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/pagination": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/console": "~5.0|~6.0|~7.0|^8.0",
+    "illuminate/filesystem": "~5.0|~6.0|~7.0|^8.0",
     "prettus/laravel-validation": "~1.1|~1.2",
```

Comment:
https://github.com/andersao/laravel-validator/pull/44

Reply:
also this is not required and much better to move in suggest, since we did not use this in any projects

Reply:
Separated PR https://github.com/andersao/l5-repository/pull/734 will apply this